server/base/src/main/java/org/apache/accumulo/server/metadata/iterators/SetEncodingIterator.java

```diff
@@ -139,6 +161,7 @@ public boolean hasTop() {
   public void init(SortedKeyValueIterator<Key,Value> source, Map<String,String> options,
       IteratorEnvironment env) throws IOException {
     this.source = source;
+    this.concat = Boolean.parseBoolean(options.getOrDefault(CONCAT_VALUE, Boolean.toString(false)));
```

Comment:
This option should always be set, so could throw an exception if its not present.  Following would throw an NPE, could do something else that creates an exception w/ a nice error message.

```suggestion
    this.concat = Boolean.parseBoolean(options.get(CONCAT_VALUE));
```

Reply:
Added in a2413f5